data-error/src/lib.rs

```diff
@@ -15,6 +15,8 @@ pub enum ArklibError {
     Parse,
     #[error("Networking error")]
     Network,
+    #[error("Storage error: {0} {1}")]
+    Storage(String, String),
```

Comment:
Why do we need 2 strings in this error type?
Could you add a doc comment to explain what they represent?